src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1242,23 +1242,12 @@ limitations under the License.
                 <requireAttribute>CLIENT_LIST</requireAttribute>
                 <requireAttribute>PARTS_LIST</requireAttribute>
             </include>
-            <include cluster="Binding" client="true" server="false" clientLocked="false" serverLocked="true">
+            <include cluster="Binding" client="false" server="false" clientLocked="true" serverLocked="true">
```

Comment:
Why we we listing it at all, then?

Please create one PR per device type so they can be reviewed separately....

Reply:
I think it's OK to have a mixed PR here, however, are the changes correct?

Reply:
They are not correct, at first glance, and in the past when trying to fix up these sorts of changes we run into issues with some things being fixed and others being broken at the same time, leading to a lot of back-and-forth and failure to converge on correct changes.

Breaking up the PR is a suggestion to mitigate that....

I can try to work with the un-broken-up thing, but I can't commit to more than two rounds of review if we do it that way.